_component_design/buttons.md

```diff
@@ -8,7 +8,7 @@ usage: >
 preview-image: preview-images/buttons.svg
 status: stable
 resource: true
-last-modified: 2018-08-27
+last-modified: 2018-09-27
 helix-ui-css: true
 helix-ui-javascript: na
 pagelink: https://rackerlabs.github.io/helix-ui/components/buttons/#basic-button
```

Comment:
On line 29, please change `site.baseURL` to site.baseurl` while we're in here.

Reply:
Done.